src/components/Dropdown/Dropdown.js

```diff
@@ -6,6 +6,7 @@ import Icon from '../Icon';
 
 export default class Dropdown extends PureComponent {
   static propTypes = {
+    ariaLabel: PropTypes.string,
```

Comment:
Should this be required?

Reply:
I set a basic one by default, but you're right we should also make it required